tests/functional/backward_compatibility/versions.txt

```diff
@@ -13,3 +13,5 @@
 0.2.1
 0.2.3
 0.2.4.post1
+0.2.5
+0.2.6
```

Comment:
0.2.6 does not exist yet. It will be added automatically after the release.

Reply:
done